README.md

```diff
@@ -27,14 +27,17 @@ $ gh repo-export --help
 Bulk exports a list of Git repositories from an organization
 
 USAGE
-  gh-repo-export [options] <organization> <repo> ...
+  gh-repo-export [options] <organization> <repo1> <repo2> ...
 
 FLAGS
   -d, --debug                         Enable debugging
       --exclude-attachments           Indicates attachments should be excluded from the migration
       --exclude-releases              Indicates releases should be excluded from the migration
       --exclude-owner-projects        Indicates projects owned by the organization or users should be excluded from the migration
+      --exclude-metadata              Indicates metadata should be excluded from the migration
+      --exclude-git-data              Indicates git data should be excluded from the migration
```

Comment:
@joshjohanning : would you be opposed to changes necessary to keep flags alphabetical?

```suggestion
      --exclude-git-data              Indicates git data should be excluded from the migration
      --exclude-metadata              Indicates metadata should be excluded from the migration
      --exclude-owner-projects        Indicates projects owned by the organization or users should be excluded from the migration
      --exclude-releases              Indicates releases should be excluded from the migration
```

I know this involves some code changes too, but I firmly believe this is important to product experience.